model/pyproject.toml

```diff
@@ -15,8 +15,26 @@ polars = "^0.20.13"
 matplotlib = "^3.8.3"
 pillow = "^10.3.0" # See #56 on CDCgov/multisignal-epi-inference
 ipykernel = "^6.29.3"
+numpydoc = "^1.6.0"
 
 
+[tool.numpydoc_validation]
+checks = [
+    "EX01",
+    "SA01",
+    "ES01",
+]
+exclude = [  # don't report on objects that match any of these regex
+    '\.undocumented_method$',
+    '\.__repr__$',
+    '\.__init__$',
+]
+override_SS05 = [  # override SS05 to allow docstrings starting with these words
+    '^Process ',
+    '^Assess ',
+    '^Access ',
+]
+
```

Comment:
Probably part of the `tool.poetry.groups.docs.dependencies` as suggested by @dylanhmorris.

Reply:
Solved in 898ac6c